.circleci/config.yml

```diff
@@ -363,6 +363,12 @@ jobs:
           paths:
             - "."
 
+  gateway:
```

Comment:
Don't think we need an additional job, can just include it under appsec

Reply:
Ack, will do